### ISSUE TITLE
[Question] DDPO Rewards almost constant throughout training (Custom Reward)

### ISSUE THREAD

**Issue by shashankg7** [huggingface/trl#2086]
Hi, 

I am finetuning diffusion model using DDPO method on a custom reward where the reward is bounded in the range [0,1]. The reward fluctuates a lot throughout training, but it stays almost at the starting value even after a couple of epochs. What could go wrong here? I am new to diffusion models, so maybe I am missing something obvious. Attaching some plots from wandb. Any suggestions here?
<img width="1366" alt="Screenshot 2024-09-19 at 9 09 17 AM" src="https://github.com/user-attachments/assets/c76b131f-5408-4573-accf-d9d684d93683">
